src/opa.js

```diff
@@ -138,12 +204,16 @@ async function _load_policy(policy_wasm, memory) {
     },
   });
 
+  /** @type {OpaInstance} */
+  env.instance = wasm.instance ? wasm.instance : wasm;
```

Comment:
I'm not sure if this was a bug or not, but this line previously would have broken if someone passed in a webassembly module as far as I could tell. See [here](https://github.com/open-policy-agent/npm-opa-wasm/pull/17/files#diff-13c3b1aeb803a65b98e91c0dca181d0cL146) for the previous version.

Reply:
Does look like it was a bug.

Can we do a type check on the `policy_wasm` parameter to make a better informed choice about how to get the instance?

Reply:
It's possible I think. I'll do a test to see if `instanceof` does the right thing. It doesn't always sadly.